programs/staking/src/lib.rs

```diff
@@ -1,314 +1,69 @@
 use anchor_lang::prelude::*;
-use anchor_spl::{associated_token::AssociatedToken, token::{CloseAccount, Mint, Token, TokenAccount, Transfer}};
 
+mod events;
```

Comment:
changing `events.rs` to `errors.rs` would require a change here as well.
```suggestion
mod errors;
```